### DISCUSSION TITLE
Always on green light for reception

### DISCUSSION THREAD

**Discussion by losehu** [egzumer/uv-k5-firmware-custom#294] [Q&A]
I replaced my K5 with a new BL24c64, which can read and write normally, but I found that it keeps turning on the green light to receive. If I flash it into the official firmware or use 1o11, it won't work like this. I'm curious why this is

**Comment by losehu** [comment]
This is not related to the level of restlessness. I adjusted it to 9, but he still keeps receiving it

**Comment by egzumer** [comment]
From what you described I can't tell anything. I have no idea.

**Comment by losehu** [comment]
![图片](https://github.com/egzumer/uv-k5-firmware-custom/assets/52628935/5ee54fd1-96e2-4dfd-acc6-e663532d78bf)
The problem lies in here (app/app.c)

**Comment by losehu** [comment]
https://github.com/egzumer/uv-k5-firmware-custom/assets/52628935/3a367d6c-a95d-48fa-89ca-cc4618b67565

I am certain that there is no signal at 400MHz. I opened SQL=9, but it is still RX. When I long press 1 or press up and down to switch frequencies, the problem disappears. However, after resetting, the problem reappears

**Comment by losehu** [comment]
I didn't press the listening button throughout the entire process, he would listen on his own

**Comment by losehu** [comment]
> 从你的描述来看我什么也看不出来。 我不知道。
Thank you very much for your work!!

**Comment by egzumer** [comment]
The BK chip detects some signal, that is all I can tell. It doesn't happen on my radio.

**Comment by losehu** [comment]
> BK 芯片检测到一些信号，这就是我所能知道的。 我的收音机上不会发生这种情况。

Yes, it's strange. My only change was to replace BL24C64
